src/bentoml/_internal/cloud/deployment.py

```diff
@@ -214,8 +214,9 @@ def update(
         if expose_endpoint is not None:
             new_config["enable_ingress"] = expose_endpoint
 
-        if api_server_config is not None and runners_config is not None:
-            api_server_config.runners = runners_config
+        if api_server_config is not None:
+            if runners_config is not None:
+                api_server_config.runners = runners_config
```

Comment:
why change this? this is the same as the previous change?

Reply:
Line 220 and 221 should be run regardless if `runners_config` is none. Previously it will only apply the changes when both `api_server_config` and `runners_config` are not none.

Reply:
but it is the same aint it?

Reply:
oh i see now